docs/step-by-step/kubeapps-on-tkg/step-1.md

```diff
@@ -1,78 +1,91 @@
+
+    > **NOTE**: Although some of the options described above use namespaces or group names containing `tmc`, this is only required for technical reasons. There is no requirement to use TMC when running Kubeapps on TKG.
+
+2. Save the `yaml` file generated at the end of the previous step.
+3. Apply this file to the cluster:
+
+  ```bash
+  kubectl apply -f pinniped-0.8-tmc.yaml
+  ```
+
+4. Confirm that Pinniped has been installed and the image is correct:
 
-At this point, you will have a TKG cluster with a compatible Pinniped version (e.g., `0.8.0`) up and running. Next, the next section will guide you configuring it to trust your favorite OIDC provider.
+```bash
+kubectl get deploy/pinniped-concierge-0-8-0 -n pinniped-concierge-0-8-0 -oyaml | grep image
+```
+
+At this point, a TKG cluster with a compatible Pinniped version is up and running.
 
-## 1.2 - Configure an OIDC Provider
+### Step 1.2: Configure an OIDC Provider
 
-In this section, we configure an OIDC provider that will authenticate users in our TKG cluster as well as in the Kubeapps dashboard. We will use the [VMware Cloud Services Platform (CSP)](https://console.cloud.vmware.com/) as the running example; nonetheless, any OIDC-compliant provider (such as Google Cloud, Azure Active Directory, Dex, Okta, etc. ) can be also configured.
-Please refer to the Kubeapps documentation on [using an OAth2/OIDC provider](https://github.com/kubeapps/kubeapps/blob/master/docs/user/using-an-OIDC-provider.md) for further information.
+The next step is to configure an OIDC provider and then configure Pinniped to trust the OIDC provider. The OIDC provider is responsible for authenticating users in the TKG cluster and the Kubeapps dashboard. The steps below use the [VMware Cloud Services Platform (CSP)](https://console.cloud.vmware.com/) as an example; however, a similar process applies to any other OIDC-compliant provider, including Google Cloud, Azure Active Directory, Dex, Okta, and others.
 
-### Create an OAuth2 Application inside CSP
+#### Create an OAuth2 Application
 
-We need to create an OAuth2 application in order to retrieve the information required by Pinniped and, later on, Kubeapps. To do so, follow the following steps:
+Begin by creating an OAuth2 application to retrieve the information required by Pinniped and, later on, Kubeapps. Follow the steps below:
 
 > **NOTE**: You must have _Developer_ access in the organization to perform these steps.
 
-1. Go to the CSP Console available at [https://console.cloud.vmware.com](https://console.cloud.vmware.com/).
-2. Under the _Organization_ settings of the right menu, click on _View Organization_
+1. Navigate to the CSP Console at [https://console.cloud.vmware.com](https://console.cloud.vmware.com/).
+2. Click the drop-down menu in the top-right corner.
+3. Under the _Organization_ settings, click _View Organization_.
    ![View organization](./img/step-1-1.png)
```

Comment:
There is something odd here where images are appearing in the wrong place in the github preview. Could you check why? It does not seem to happen in Step 4, but only here.

Reply:
mmm, that's odd. I can replicate it, it's a matter of the spaces used to indent the image I think. Let me check, I'll take care of it after merging